.eslintrc.js

```diff
@@ -3,14 +3,14 @@ module.exports = {
   "env": {
     "mocha": true,
     "node": true,
+    "browser": true,
   },
   "plugins": ["chai-friendly"],
   "rules": {
-    "no-unused-expressions": 0,
     "chai-friendly/no-unused-expressions": 2,
-    "browser": true,
     "import/no-extraneous-dependencies": 0,
     "import/extensions": 0,
     "import/no-unresolved": 0,
+    "jsx-quotes": [2, "prefer-single"],
```

Comment:
`2` is the same thing as `"error"`, `1` is the same thing as `"warn"`, and `0` is the same thing as `"off"`. If you enter an invalid value you'll get this error: `Severity should be one of the following: 0 = off, 1 = warn, 2 = error`.